mkdocs.yml

```diff
@@ -4,6 +4,9 @@ nav:
   - Components: components.md
   - Contributing: CONTRIBUTING.md
   - Community: community.md
+  - News:
+    - Releases:
+      - QMCPy v2.1: releases/QMCPy-2.1-announcement.md
```

Comment:
I recommend we delete this and the QMCPy-2.1-announcement.md file after submitting our announcement to whichever external website we chose. See review comment